demisto_sdk/commands/format/format_module.py

```diff
@@ -89,7 +90,11 @@ def format_manager(input: str = None,
         files = get_files_in_dir(input, ['json', 'yml', 'py'])
     else:
         files = [file['name'] for file in
-                 get_changed_files(filter_results=lambda _file: not _file.pop('status') == 'D')]
+                 get_changed_files(from_branch='origin/master',
```

Comment:
@roysagi this change solved an issue but may cause problems for contributors

Reply:
Perhaps we can take the branch as an optional parameter.
Not the prettiest solution but doable @yaakovi @roysagi